src/storage/txn/commands/prewrite.rs

```diff
@@ -1511,6 +1511,64 @@ mod tests {
         }
     }
 
+    // this test for prewrite with should_not_exist flag
+    #[test]
+    fn test_prewrite_should_not_exist() {
```

Comment:
You should be able to get the current `max_ts` from `cm.max_ts()`. How about also checking the exact value in the tests?

Reply:
Done.
PTAL